Lucidtech/Las/Client.cs

```diff
@@ -364,12 +364,14 @@ public object CreateBatch(string? name = null, string? description = null)
             string documentId,
             string modelId,
             int? maxPages = null,
-            bool? autoRotate = null
+            bool? autoRotate = null,
+            string? imageQuality = null
```

Comment:
Not Enum?

Reply:
We don't have enums for the other "enum-like" variables like transitionType and order, so i suggest we take that in another PR for consistency